src/QsFmt/App/CommandLineArguments.fs

```diff
@@ -178,22 +243,18 @@ type UpdateAndFormatArguments =
                 )
         }
 
-type CommandKind =
-    | Update
-    | Format
-    | UpdateAndFormat
-
-type Arguments =
+type CommandWithOptions =
     {
-        CommandKind: CommandKind
+        CommandKind: CommandKind option
+        InputKind: InputKind option
```

Comment:
I'm not quite seeing the need for these to be option types. Both of these need to have values and have default values when none are given.

Reply:
This is for the case in which we didn't have a chance to actually get a valid Command or Input type due to bad arguments